kwok/tests/kwok-playbook.yml

```diff
@@ -0,0 +1,17 @@
+# Save the output of this file and use kubectl create -f to import
```

Comment:
Barak suggested that `playbook` is not a good name, I tend to agree (the existence of Ansible playbooks is confusing). I've gone with manifest instead, but do tell if you have better suggestions.